deploy/olm-catalog/3scale-operator-master/3scale-operator-master.v0.0.0.clusterserviceversion.yaml

```diff
@@ -14,82 +14,955 @@ metadata:
-    - kind: Binding
-      name: bindings.capabilities.3scale.net
-      version: v1alpha1
-      description: Binding
-      displayName: Binding
-    - kind: Limit
-      name: limits.capabilities.3scale.net
-      version: v1alpha1
-      description: Limit
-      displayName: Limit
-    - kind: MappingRule
-      name: mappingrules.capabilities.3scale.net
-      version: v1alpha1
-      description: MappingRule
-      displayName: MappingRule
-    - kind: Metric
-      name: metrics.capabilities.3scale.net
-      version: v1alpha1
-      description: Metric
-      displayName: Metric
-    - kind: Plan
-      name: plans.capabilities.3scale.net
-      version: v1alpha1
-      description: Plan
-      displayName: Plan
-    - kind: Tenant
-      name: tenants.capabilities.3scale.net
-      version: v1alpha1
-      description: Tenant
-      displayName: Tenant
+      - kind: APIManager
+        name: apimanagers.apps.3scale.net
+        version: v1alpha1
+        description: API Manager
+        displayName: API Manager
+        resources:
+          - kind: DeploymentConfig
+            version: apps.openshift.io/v1
+          - kind: PersistentVolumeClaim
+            version: v1
+          - kind: Service
+            version: v1
+          - kind: Route
+            version: route.openshift.io/v1
+          - kind: ImageStream
+            version: image.openshift.io/v1
+        specDescriptors:
+          - description: Image Stream Tag Import Insecure
+            displayName: Image Stream Tag Import Insecure
+            path: imageStreamTagImportInsecure
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+          - description: Resource Requirements Enabled
+            displayName: Resource Requirements Enabled
+            path: resourceRequirementsEnabled
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+          - description: Wildcard domain as configured in the API Manager object
+            displayName: Wildcard Domain
+            path: wildcardDomain
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:text'
+          - description: MySQL Image
+            displayName: Image
+            path: system.database.mysql.image
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:system'
+              - 'urn:alm:descriptor:com.tectonic.ui:text'
+          - description: PostgresSQL Image
+            displayName: Image
+            path: system.database.postgresql.image
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:system'
+              - 'urn:alm:descriptor:com.tectonic.ui:text'
+          - description: AWS Bucket
+            displayName: AWS Bucket
+            path: system.fileStorage.amazonSimpleStorageService.awsBucket
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:system'
```

Comment:
Are nested fieldGroups configurable? Could this be placed inside the "amazonSimpleSotrageService" group?
How it looks currently from one of the captures provided this field appears under the .spec.system section but it's really under the .spec.system.fileStorage.amazonSimpleStorageService section.

Same situation applies to several other fields that are nested on nested fields.

Reply:
Yes, "nested fieldgroups" are not yet implemented. Even the component to add and delete into object arrays is not implemented, so as of now I have just added one element for it.
e.g " integrationMethod.apicastHosted.mappingRulesSelector.matchExpressions[0].key" (line #323 and many others).